lefthook.yml

```diff
@@ -1,6 +1,6 @@
 pre-commit:
   parallel: true
   commands:
-    eslint:
-      glob: "*.{js,ts,jsx,tsx}"
-      run: yarn lint {staged_files}
+    lintjavascript:
+      glob: "*.{js,ts,tsx}"
```

Comment:
This fixes a problem with lefthook where it could 
not find the pages directory.